templates/shortcode-receipt.php

```diff
@@ -150,12 +150,12 @@
 						<div class="edd_purchase_receipt_product_name">
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
-							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
+							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_attr( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
 							<?php endif; ?>
 						</div>
 
 						<?php if ( $edd_receipt_args['notes'] ) : ?>
-							<div class="edd_purchase_receipt_product_notes"><?php echo wpautop( edd_get_product_notes( $item['id'] ) ); ?></div>
+							<div class="edd_purchase_receipt_product_notes"><?php echo wp_kses_post(wpautop( edd_get_product_notes( $item['id'] ) ) ); ?></div>
```

Comment:
@mindctrl thanks.   I realized my last post I was trying with $allowedposttag and not $allowedposttags on accident.  I went ahead and tried your suggestions.

I now have 

line 5
```
 global $edd_receipt_args, $allowedposttags;
```

and 

line 158
```
<?php echo strip_tags( edd_get_product_notes( $item['id'] ), $allowedposttags ); ?>
```

I had to take out the semi colon right after $allowedposttags since it threw syntax error.   I get the following error now.

Notice: Array to string conversion in /var/www/html/wp-content/plugins/easy-digital-downloads/templates/shortcode-receipt.php on line 158